packages/commonwealth/server/api/user.ts

```diff
@@ -3,4 +3,5 @@ import { User } from '@hicommonwealth/model';
 
 export const trpcRouter = trpc.router({
   updateUser: trpc.command(User.UpdateUser, trpc.Tag.User),
+  getCommunitiesWithNewContent: trpc.query(User.GetNewContent),
```

Comment:
We are trying to use the same name, in this case if the endpoint is `getCommunitiesWithNewContent`, the query should be named accordingly - `GetCommunitiesWithNewContent`

Reply:
didn't knew, thanks, will update tomorrow.